File: app/src/main/java/com/mgaetan89/showsrage/activity/HistoryActivity.java
package com.mgaetan89.showsrage.activity;

import android.support.v4.app.Fragment;

import com.mgaetan89.showsrage.R;
import com.mgaetan89.showsrage.fragment.HistoryFragment;

public class HistoryActivity extends BaseActivity {
	@Override
	protected boolean displayHomeAsUp() {
		return false;
	}

	@Override
	protected Fragment getFragment() {
		return new HistoryFragment();
	}

	@Override
	protected int getSelectedMenuId() {
		return R.id.menu_history;
	}

	@Override
	protected int getTitleResourceId() {
		return R.string.history;
	}
}

File: app/src/main/kotlin/com/mgaetan89/showsrage/activity/HistoryActivity.kt
package com.mgaetan89.showsrage.activity

import com.mgaetan89.showsrage.R
import com.mgaetan89.showsrage.fragment.HistoryFragment

class HistoryActivity : BaseActivity() {
    override fun displayHomeAsUp() = false

    override fun getFragment() = HistoryFragment()

    override fun getSelectedMenuId() = R.id.menu_history

    override fun getTitleResourceId() = R.string.history
}
